wls-ergebnismeldung-service/src/main/java/de/muenchen/oss/wahllokalsystem/ergebnismeldungservice/client/DummyClientImpl.java

```diff
@@ -0,0 +1,27 @@
+package de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.client;
+
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.configuration.Profiles;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.awerte.AWerteClient;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.awerte.AWerteModel;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.WlsException;
+import de.muenchen.oss.wahllokalsystem.wls.common.security.domain.BezirkUndWahlID;
+import java.util.List;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile(Profiles.DUMMY_CLIENTS)
+public class DummyClientImpl
+        implements AWerteClient {
+
+    @Override
+    public List<AWerteModel> getAWerte(final String wahlbezirkID) throws WlsException {
+        if (wahlbezirkID.equals("wahlbezirkID")) {
```

Comment:
finde ich schon eine sehr versteckte Magie an die man denken muss wenn man mit dem Client arbeitet.

Ich fände es besser wenn es eine Allow oder Blicklist in der Config für den Dummy client gäbe. Dann muss man sich nicht durch den Code wühlen warum mal was kommt vom Client und wann nicht.

Reply:
Ich habe den Client vereinfacht, er gibt einfach immer einen AWert zurück da im Service ja sicher gestellt wird, dass ein Aufruf mit ungültiger wahlbezirkId nicht statt findet.

resolved with 52bb8d97508d0202393a33147f19922fb3a485ba